DEPS.bzl

```diff
@@ -6883,6 +6875,14 @@ def go_deps():
             "https://storage.googleapis.com/cockroach-godeps/gomod/github.com/pierrec/lz4/com_github_pierrec_lz4-v2.6.0+incompatible.zip",
         ],
     )
+    go_repository(
```

Comment:
I wonder if we really need both of this (maybe we can update alll code to use v4)?

Reply:
Looks like only one place for the old package, and recent: https://github.com/shermanCRL/cockroach/blob/master/pkg/sql/sem/builtins/compression.go cc @miretskiy

Reply:
I’ve removed the older one.